tests/services/detectionlists/test_departing_employee.py

```diff
@@ -107,6 +108,18 @@ def mock_py42_response(self, mocker, mock_get_case_details_function, py42_respon
 
         return py42_response
 
+    @pytest.fixture
+    def mock_detection_list_user_client_failure(
```

Comment:
What kind of failure?

Should be `mock_detection_list_user_not_found_client`

Reply:
Removed.